tutorials/users/ide/version-control.md

```diff
@@ -1 +1,607 @@
-# version-control
+# Version Control
+
+[_Version control_](https://en.wikipedia.org/wiki/Version_control) is a powerful way to organize, back up,
+and share your research outputs and methodologies (code + data) with collaborators, or the public at large.
+A version control system keeps track of a set of files and saves snapshots (i.e. _versions_, _commits_) of 
+the files at any point in time. Using version control allows you to confidently make changes to your code 
+(and any other files), with the ability to roll back to any previous state.
+
+
+## Introduction to Git
+
+Git is one of the most popular version control systems in use today. It has a native command line interface but there 
+are also many third party desktop GUI clients that help make it easier to get started with Git.  A comprehensive list of 
+several freely available Git clients is [available here](https://git-scm.com/downloads/guis).
+
+Online Git services like [GitHub](https://github.com/), [GitLab](https://about.gitlab.com/), 
+or [BitBucket](https://bitbucket.org/) allow you to store your repository online and access it 
+anywhere with a free account. In this tutorial, you'll learn how to set up a remote Git repository to backup and share 
+your work using GitHub, and how to configure Git locally, within your Marble Jupyter Lab environment to use that account.
+
+(github)=
+## GitHub
+
+GitHub is one of the many online providers of free remote repository storage.  This section will cover how to work with 
+remote repositories in a Github context.
+
+### Creating a GitHub Account
+To begin using Git you need to create a free account on GitHub. Go to the [GitHub](https://github.com/) website and follow the instructions to create an account. 
```

Comment:
That's a good point. I largely left here what Alex had previously because I wanted to bring something up for discussion with you both (@alexandercyu). I think that Github section should go after Git. I think talking about online repositories (which we ourselves acknowledge in the document that it is an optional step), before setting up a local repo is premature. 

Thoughts?

Reply:
I agree

Reply:
Moved the github section to the bottom after all the git sections